backend/config.js

```diff
@@ -23,10 +24,10 @@ var environments = {
             expiresAfterSeconds: 360000 * 24 // 24 hour
         },
         pgConnect: {
-            user: process.env.RDS_USERNAME || process.env.INDABA_PG_USERNAME || 'db_user',
+            user: process.env.RDS_USERNAME || process.env.INDABA_PG_USERNAME || 'indabauser',
             testuser: process.env.RDS_TESTUSER || process.env.INDABA_PG_TESTUSER || 'test', // make trust method for this user in PostgreSQL Client Authentication Configuration File (pg_hba.conf)
             password: process.env.RDS_PASSWORD || process.env.INDABA_PG_PASSWORD || 'password',
-            database: process.env.INDABA_PG_DB || 'database',
+            database: 'indabatestuser' || process.env.INDABA_PG_DB || 'indaba',
```

Comment:
We want to set this in the .env file instead from here on. The reason I was setting it before locally was because I didn't realize I had *already* set my env variable to be something else.